app/gen-server/lib/HomeDBManager.ts

```diff
@@ -1953,15 +1953,20 @@ export class HomeDBManager extends EventEmitter {
   // Update the webhook url in the webhook's corresponding secret (note: the webhook identifier is
   // its secret identifier).
   public async updateWebhookUrlAndAuth(
-    id: string, docId: string, url: string, auth: string | undefined, outerManager?: EntityManager) {
+    id: string, docId: string, url: string | undefined, auth: string | undefined, outerManager?: EntityManager) {
     return await this._runInTransaction(outerManager, async manager => {
       const value = await this.getSecret(id, docId, manager);
       if (!value) {
         throw new ApiError('Webhook with given id not found', 404);
       }
       const webhookSecret = JSON.parse(value);
-      webhookSecret.url = url;
-      webhookSecret.authorization = auth;
+      // update url and authorization only if not undefined to fit to patch calls
```

Comment:
If the user wants to empty the values of the url or of the auth, can s/he do that?

Reply:
If that works as I suspect (the user may empty the values, the client sends empty strings), 

```suggestion
      // As we want to patch the webhookSecret object, only set the url and the authorization when they are defined.
      // When the user wants to empty the value, we are expected to receive empty strings.
```